charts/squid/templates/squid-deployment.yaml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: squid-service
+  labels:
+    app: squid
+spec:
+  ports:
+  - port: 3128
+    name: squid
+  - port: 9301
+    name: exporter
```

Comment:
Its okay to expose the exporter with the service but keep in mind that you should be using service discovery to scrape the squid pod(s). If you have multiple pods, the service will proxy you to a random one each time.